nncf/tensorflow/pruning/base_algorithm.py

```diff
@@ -300,6 +300,31 @@ def _check_pruning_level(self, params):
         if pruning_flops_target:
             self.prune_flops = True
 
+    def _calculate_num_of_sparse_elements_by_node(self) -> Dict[str, int]:
+        num_of_sparse_elements_by_node = {}
+        wrapped_layers = collect_wrapped_layers(self._model)
+        for wrapped_layer in wrapped_layers:
+            for weight_attr, ops in wrapped_layer.weights_attr_ops.items():
+                for op_name in ops:
+                    if op_name in self._op_names:
```

Comment:
I think you can use `get_nncf_operations()` method from the `nncf.tensorflow.graph.utils` module. Could you please look at this?

Reply:
Thanks for the suggestion. Indeed I can, used.